src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  box-sizing: border-box;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    display: block;
+
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $textColorSecondary;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin: 0;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    margin: 0;
+    color: $textColorSecondary;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $textColorSecondary;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    font-weight: 700;
+    font-size: 14px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #fff;
+    text-decoration: none;
+
+    background-color: $colorMain;
+    border: 1px solid $colorMain;
+    border-radius: 5px;
+
+    transition: color 0.3s, background-color 0.3s;
+
+    &:hover {
+      color: $colorMain;
+      background-color: #fff;
```

Comment:
I see you have used color #fff three times in the card block, isn't it better to use it as a variable as you did for $colorMain?